edge-modules/MetricsCollector/MetricsScrapeAndUpload.cs

```diff
@@ -59,9 +57,16 @@ IEnumerable<Metric> GetGuidTaggedMetrics(IEnumerable<Metric> metrics)
         {
             foreach (Metric metric in metrics)
             {
-                Dictionary<string, string> tagsWithGuid = new Dictionary<string, string>(metric.Tags);
-                tagsWithGuid.Add("guid", this.metricsCollectorRuntimeId.ToString());
```

Comment:
why remove this tag for this.metricsCollectorRuntimeId?

Reply:
I renamed the tag name to testId